cdc_rsync/params.cc

```diff
@@ -323,6 +324,8 @@ OptionResult HandleParameter(const std::string& key, const char* value,
 
 bool ValidateParameters(const Parameters& params, bool help) {
   if (help) {
+    std::cout << "cdc_rsync - Client component. Version: " << BUILD_VERSION
```

Comment:
"Client component" is not a good description for the tool. Note that this is the first thing that the user sees. Change to "Synchronize files and directories" and remove that above, so that the full help reads:

```
cdc_rsync - Synchronize files and directories. Version: 1.23

Matching files are skipped based on file size and modified time. For partially
matching files only the differences are transferred. The destination directory
can be the same Windows machine or a remote Windows or Linux device.

Usage:
  cdc_rsync [options] source [source]... [[user@]host:]destination
...
```